lte/gateway/c/session_manager/ShardTracker.cpp

```diff
@@ -13,45 +13,53 @@
 
 #include <string>
 #include <vector>
+#include <set>
 
 #include "ShardTracker.h"
 
 namespace magma {
 
-int ShardTracker::add_ue() {
-  /*
-  * If there are no shards, add an entry for the shard
-  * representing a single UE at the new shard
-  */
-  if (ue_count_per_shard_.size() == 0) {
-    ue_count_per_shard_.push_back(1);
-    return 0;
+int ShardTracker::add_ue(std::string imsi) {
+  if (shards_.empty()) {
+    std::set<std::string> new_shard = {imsi};
+    shards_.push_back(new_shard);
   }
-  /*
-  * Iterate through all existing shards, if all are full
-  * create a new shard with quantity 1, otherwise increment
-  * the quantity of UEs in the latest shard
-  */
-  for (size_t shard_id = 0; shard_id < ue_count_per_shard_.size(); shard_id++) {
-    if (ue_count_per_shard_[shard_id] < max_shard_size_) {
-      ue_count_per_shard_[shard_id]++;
+  for (size_t shard_id = 0; shard_id < shards_.size(); shard_id++) {
+    /**
```

Comment:
Did we decide on using `//` for multi line comments in the last PR? Or `/**`? (Just making sure, I can't remember which :) )

Reply:
Changed to // comments for multilines(except for function docs).